hazelcast-jet-reference-manual/src/main/asciidoc/work-with-jet/log-debug.adoc

```diff
@@ -178,9 +178,16 @@ Additionally, if the vertex sourcing them is a data source or data sink, then th
 |===
 |Names|Extra tags
 
+|**executionStartTime**: Start time of the current execution of the job. +
+**executionCompletionTime**: Start time of the current execution of the job. +
+
+Both contain epoch time in milliseconds.
+|** no extra tags, they are even missing `vertex`** +
```

Comment:
`they are even missing 'vertex'` - no surprise here. I personally would just skip this part, the next sentence is enough,